README.md

```diff
@@ -43,7 +43,7 @@ From Local:
     a) ./localDocker.sh (from root directory)
     b) docker run -p 80:80 hub:local
    
-2) set base.url property in integration-hub.properties
+2) set base.url property in functional-hub.properties
```

Comment:
The file might already be `system-hub.properties`, but I think that's the name we should use. I think the rest of this readme may need review as well.